lib/orbit/lib/diffs.js

```diff
@@ -91,6 +91,10 @@ var diffs = function(a, b, options) {
               bi++;
             }
           }
+        } else if (a instanceof Date) {
+          if (a.getTime() === b.getTime()) return;
```

Comment:
Thanks for the info, seems "covered"! It was really unclear from looking at the diff!